app/components/Home/Yaroslav_Grushchak/Projects/Project_1/components/header/header.component.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+// import './header.component.css';
+import './header.component.styl';
+import { HashLink as Link } from 'react-router-hash-link';
+
+const Header = () => (
+    <header className="container-fluid">
+        <div className="container-fluid top_nav">
+            <div className="container">
+                <a href="#" className="logo">
+                    <img src={require("../../images/logo.png")} alt="Blueasy"/>
```

Comment:
You can set src without require func

Reply:
The image isn't displayed without require func for me.